src/aap_eda/tasks/analytics.py

```diff
@@ -60,10 +61,30 @@ def schedule_gather_analytics(
         )
 
 
-@job(ANALYTICS_TASKS_QUEUE)
 def reschedule_gather_analytics(
     queue_name: str = ANALYTICS_TASKS_QUEUE,
 ) -> None:
+    """Reschedule the gather analytics job.
+
+    Proxy for reschedule_gather_analytics_rq and
+    reschedule_gather_analytics_dispatcherd.
+    """
+    if features.DISPATCHERD:
+        return reschedule_gather_analytics_dispatcherd(queue_name)
+    return reschedule_gather_analytics_rq(queue_name)
+
+
+@job(ANALYTICS_TASKS_QUEUE)
```

Comment:
We don't need this decoration, do we?

Reply:
good catch. https://github.com/ansible/eda-server/pull/1317